### ISSUE TITLE
[$15][MSFT-56] [Registration] A heading level is skipped "Username Tips".

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2773]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Inspect Element
3. Observe the heading level in User Name Tips.

**Expected Result**
A heading level should not be skipped.

**Actual Result**
A heading level is skipped.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AAA
**WCAG Success Criterion**: 2.4.10 Section Headings
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096314 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096314 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096314 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
Already fixed in this PR: https://github.com/appirio-tech/accounts-app/pull/239/files

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2773 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2773 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2773 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097501 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097501 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by ajefts** [comment]
Passed in dev.

**Comment by nkumar-topcoder** [comment]
@drasticdpk @tosha5252 @lakshmiathreya this issue is same as #2812 PR 239 and it is already pushed to prod, verified and closed. Please reverify in prod.

**Comment by tosha5252** [comment]
Veri
![Annotation 2019-07-29 231827](https://user-images.githubusercontent.com/6400904/62100415-4c173900-b257-11e9-80b6-0eadf898eb02.png)
fied on Chrome, WIndows 10

**Comment by tosha5252** [comment]
C

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097501<br/><br/>```This is an automated message for crazyk via Topcoder X```
